week4/src/components/DetailWeekInfo.jsx

```diff
@@ -0,0 +1,48 @@
+import { useParams } from "react-router-dom";
+import ImgCard from "./ImgCard";
+import styled from "styled-components";
+import ErrorPage from "./ErrorPage";
+import useWeekFetch from "../hooks/useWeekFetch";
+
+const DetailWeekInfo = () => {
+  const params = useParams();
+  const { cityId } = params;
+
+  const [detailCardList, isLoading, isError] = useWeekFetch(
+    `https://api.openweathermap.org/data/2.5/forecast?q=${cityId}&appid=${
+      import.meta.env.VITE_APP_WEATHER
+    }&units=metric`
+  );
+
+  return (
+    <CardListWrapper>
+      {(detailCardList.length !== 0 && !isError) || !isError ? (
+        detailCardList &&
```

Comment:
```suggestion
    {detailCardList && !isError ? (
```
요기도 요렇게만 쓰면 안되나용/!? 몬가 조건들이 겹치는 것 같아서!!

Reply:
마자!! 진짜 보다보니까 조건들이 겹치구 있어서 현지가 말한 것처럼 다 수정해줬어! 고마왕><